regolith/helpers/makeappointmentshelper.py

```diff
@@ -17,18 +17,26 @@
 from regolith.tools import (
     all_docs_from_collection,
     get_pi_id,
+    fuzzy_retrieval,
+    is_fully_appointed
+)
+from regolith.dates import (
+    is_current,
+    has_finished,
 )
-from regolith.dates import is_current
 
 TARGET_COLL = "people"
 HELPER_TARGET = "makeappointments"
 
 
 def subparser(subpi):
 
+    subpi.add_argument("--date", help="date to check appointments and grants against")
     subpi.add_argument("-v", "--verbose", action="store_true", help='increase verbosity of output')
     subpi.add_argument("-g", "--gap", action="store_true",
                        help='get group members with a gap in their appointments')
+    subpi.add_argument("-b", "--begin_date", help='begin date of interval to check appointments')
+    subpi.add_argument("--duration", help='duration of interval to check appointments', default=1)
     subpi.add_argument("-o", "--out_of_date", action="store_true",
```

Comment:
I don't think I need the -o flag or the -g flag because I can't ever think of a time when I don't want to know this info (also which grant went out of date)

Reply:
should I delete the --depleted flag as well (for grant out of money)?

Reply:
yes, I think delete

Reply:
done